docs/sources/guides/basic_concepts.md

```diff
@@ -66,7 +66,7 @@ There are a wide variety of styling and formatting options that each Panel expos
 
 Panels can be dragged and dropped and rearranged on the Dashboard. They can also be resized.
 
-There are currently five Panel types: [Graph](/reference/graph/), [Singlestat](/reference/singlestat/), [Dashlist](/reference/dashlist/), [Table](/reference/table_panel/), and [Text](/reference/text/).
+There are currently eight Panel types: [Graph](/reference/graph/), [Logs](/reference/logs/), [Singlestat](/reference/singlestat/), [Table](/reference/table_panel/), [Heatmap](/reference/heatmap/), [Alert list](/reference/alertlist/), [Dashboard list](/reference/dashlist/), and [Text](/reference/text/).
```

Comment:
```suggestion
These are the available Panel types: 

- [Alert list](/reference/alertlist/)
- [Dashboard list](/reference/dashlist/)
- [Graph](/reference/graph/)
- [Heatmap](/reference/heatmap/)
- [Logs](/reference/logs/)
- [Singlestat](/reference/singlestat/)
- [Table](/reference/table_panel/)
- [Text](/reference/text/)
```

I think this would do better as a alphabetically ordered list. Especially with the added types. Also, I don't think it's necessary to document how many there are, seeing we list them all.